### ISSUE TITLE
l'énigme "les 3 singes" ne se réinitialise pas

### ISSUE THREAD

**Issue by SteeveDroz** [CPLN/enigmos#186]
Lorsqu'on a déjà commencé l'énigme "Les 3 Singes" et qu'on passe, elle ne se réinitialise pas. Elle est à nouveau dans le même état lorsqu'on revient dessus.

**Comment by SteeveDroz** [comment]
@AnaKamelia, vous voulez vous en occuper ?

**Comment by AnaKamelia** [comment]
Oui, ça m'occupera cet après midi. Je précise aussi par écrit que c'est cette énigme que je souhaite soumettre à l'évaluation pour la note d'atelier.
